test/fixtures/skeleton-lite/package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
adding this file is necessary - yarn simply creates empty node_modules and lockfile, but pnpm install would raise an error here.